python/prima/__init__.py

```diff
@@ -60,9 +60,9 @@ def process_constraints(constraints, x0, options):
             raise ValueError("Constraint type not recognized")
 
     if len(nonlinear_constraints) > 0:
-        nonlinear_constraint = process_nl_constraints(x0, nonlinear_constraints, options)
+        nonlinearconstraint_function = process_nl_constraints(nonlinear_constraints)
```

Comment:
Would `nonlinear_constraint_function` be better than `nonlinearconstraint_function`?